src/CodeGen_Internal.cpp

```diff
@@ -476,5 +476,82 @@ void set_function_attributes_for_target(llvm::Function *fn, Target t) {
     fn->addFnAttr("reciprocal-estimates", "none");
 }
 
+void embed_bitcode(llvm::Module *M, const string &halide_command) {
```

Comment:
I know that this code is meant to emulate the ` -fembed-bitcode` flag of Clang; just curious if the necessary behavior is actually documented anywhere, vs just reverse-engineered from Clang. Probably worth adding some comments about that either way, with link to spec or comments about where in Clang the behavior was observed, in order to make future maintenance simpler.